codalab/client/local_bundle_client.py

```diff
@@ -843,7 +843,8 @@ def resolve_interpreted_items(self, interpreted_items):
             elif mode == 'contents':
                 info = self.get_target_info(data, 1)
                 if 'type' not in info:
-                    pass
+                    import base64
```

Comment:
This should not be inserted in local_bundle_client.py, which should in general return the raw data (in this case None representing the fact that the file doesn't exist).  This should be resolved at the caller of resolve_interpreted_items side (in the codalab repo).  Also, I think the CLI still has this problem (cl print).